posthog/temporal/feature_flag_sync/workflow.py

```diff
@@ -0,0 +1,219 @@
+                raise
+
+        # Store checkpoint for next sync
+        redis_client.set(FEATURE_FLAG_LAST_CALLED_SYNC_KEY, current_sync_timestamp.isoformat())
+
+        duration = (django_timezone.now() - start_time).total_seconds()
+
+        logger.info(
+            "Feature flag sync completed",
+            updated_count=updated_count,
+            processed_events=total_events,
+            clickhouse_results=len(result),
+            duration_seconds=duration,
+        )
+
+        # Emit metrics for successful completion
+        get_feature_flag_sync_finished_metric("completed").add(1)
+        get_feature_flag_sync_duration_metric().record(int(duration * 1000))
+        get_feature_flag_sync_events_processed_metric().add(total_events)
+        get_feature_flag_sync_flags_updated_metric().add(updated_count)
+
+        return FlagSyncResult(
+            updated_count=updated_count, processed_events=total_events, sync_duration_seconds=duration
+        )
+
+    except Exception as e:
+        # Emit metrics for failed completion
+        get_feature_flag_sync_finished_metric("failed").add(1)
+        duration = (django_timezone.now() - start_time).total_seconds()
+        get_feature_flag_sync_duration_metric().record(int(duration * 1000))
+
+        capture_exception(e)
+        raise
+
+
+@temporalio.workflow.defn(name="feature-flag-sync")
+class SyncFeatureFlagLastCalledWorkflow(PostHogWorkflow):
+    """Workflow for syncing feature flag last_called_at timestamps from ClickHouse to PostgreSQL."""
+
+    @staticmethod
+    def parse_inputs(inputs: list[str]) -> SyncFeatureFlagLastCalledInputs:
+        """Parse inputs from the management command CLI."""
+        if inputs:
+            loaded = json.loads(inputs[0])
+            return SyncFeatureFlagLastCalledInputs(**loaded)
+        return SyncFeatureFlagLastCalledInputs()
+
+    @temporalio.workflow.run
+    async def run(self, inputs: SyncFeatureFlagLastCalledInputs | None = None) -> FlagSyncResult:
+        """
+        Main workflow execution.
+
+        This workflow syncs feature flag last_called_at timestamps by:
+        1. Running the sync activity with retry policy
+        2. Returning the sync results for observability
+        """
+        if inputs is None:
+            inputs = SyncFeatureFlagLastCalledInputs()
+
+        result = await temporalio.workflow.execute_activity(
+            sync_feature_flag_last_called_activity,
+            inputs,
+            start_to_close_timeout=timedelta(minutes=10),  # Generous timeout for large datasets
+            retry_policy=temporalio.common.RetryPolicy(
+                initial_interval=timedelta(seconds=30),
+                maximum_attempts=3,
+                maximum_interval=timedelta(minutes=2),
+            ),
+        )
+
+        return result
+
+
+def get_last_sync_timestamp(redis_client, lookback_days=1) -> datetime:
```

Comment:
can we make the `lookback_days` configurable with an environment variable + default